src/autogluon_dashboard/app.py

```diff
@@ -33,7 +35,8 @@
 from autogluon_dashboard.scripts.data import get_dataframes
 from autogluon_dashboard.scripts.widget import Widget
 
-dataset_file, aggregated_file = "dev_data/all_data.csv", "dev_data/autogluon.csv"
+dataset_file = os.environ.get("PER_DATASET_S3_PATH", "dev_data/all_data.csv")
```

Comment:
Don't use relative path as the default value, you don't know where user's current working directory is. Use the `__file__` thing we used before

Reply:
Sounds good